extension/EdFi.Ods.Extensions.EdFiLMS/LMS-Harmonizer/1080-lms-assignment-schoology.sql

```diff
@@ -0,0 +1,66 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE OR ALTER PROCEDURE lms.harmonize_assignment_schoology AS
+BEGIN
+
+	INSERT INTO [edfilms].[Assignment]
+		([AssignmentIdentifier]
+		,[LMSSourceSystemDescriptorId]
+		,[Title]
+		,[AssignmentCategoryDescriptorId]
+		,[AssignmentDescription]
+		,[StartDateTime]
+		,[EndDateTime]
+		,[DueDateTime]
+		,[MaxPoints]
+		,[SectionIdentifier]
+		,[LocalCourseCode]
+		,[SessionName]
+		,[SchoolYear]
+		,[SchoolId]
+		,[CreateDate]
+		,[LastModifiedDate]
```

Comment:
These two columns have default values on them, therefore you don't need to set them when inserting.

When updating, however, the `LastModifiedDate` must be set manually.